calculator/main.py

```diff
@@ -58,18 +81,25 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    records = []
```

Comment:
You can just `return calc['history']` here instead of manually building a list as it's already being stored as one.